add some tests for themefileupload as these fail with current plone.rest

djay opened the pull request:
see https://github.com/plone/plone.rest/issues/59

There is also a bug with the testbrowser used so test not really complete

jensens commented:
@djay code-wise I am fine merging it, I have no idea why testbrowser comes in our way (again)... 

I started the tests (please do this on your own in future, it would save me lots of time). 

May you fix the mismatched user id of the commit so it matches something GitHub connects with your account?

Also a short note in the changelog is missing.

djay commented:
@jensens I added the change log. I wasn't sure I was supposed to when its just adding a test since it doesn't fit one of the 3 criteria. Given its a failing test I guess its bug fix in the making, so I put it there.

The contributor agreement I'm not sure how to solve. I've added every email I can think of to github that might have been used when I signed my contributor agreement. I'm in the plone github group (the old way of proving you signed). I did sign, a long time ago. Is there any way to find out what email it has me registered as?

jensens commented:
@djay both, `Author` and `Commit` are set to software[at]pretaweb.com - so if this address is in your GitHub account it should detect you. If it still do not detect you as contributor  something is broken and we need to ask the testing team. Can you confirm this address is set in your GitHub account?

djay commented:
@jensens weirdly that email is listed as my public profile address but wasn't in the list of verified emails. Hopefully thats fixed now

djay commented:
@jensens agreement verifier is still not working. @plone/testing-team any suggestions?

jensens commented:
A close and reopen triggers the checks again, this fixed it.

jensens commented:
@djay It seems Plone 5.2s Zope 4 thinks ``200 OK`` is better over Zope2s ``200 Ok`` (do not ask me why) - but to get around it, what about just checking for ``self.assertIn('Status: 200', str(self.browser.headers))``?

djay commented:
@jensens made a fix and running again now.
When does plone.rest get merged into the core? Thats what the test is designed to fail on.

jensens commented:
Should happen in the 5.2series. Iirc, the plip should contain a statement about this, if not we have a topic for the next FWT meeting. (at my smartphone and too lazy to look it up)

djay commented:
@jensens in that case I think this is ready to merge
